ext/hash/sha3/generic64lc/brg_endian.h

```diff
@@ -44,9 +44,7 @@
 #elif defined( __linux__ ) || defined( __GNUC__ ) || defined( __GNU_LIBRARY__ )
 #  if !defined( __MINGW32__ ) && !defined( _AIX )
 #    include <endian.h>
-#    if !defined( __BEOS__ )
-#      include <byteswap.h>
-#    endif
+#    include <byteswap.h>
```

Comment:
Same as above. Here might be even useful to remove the entire `#if 0` block with not used code here... I don't know if these are useful as comments but certainly look a bit bloated when reading the code.

Reply:
Aren’t these files a bundled lib?

Reply:
They are, yes https://github.com/XKCP/XKCP

Reply:
I think if that is the case we should just leave it instead of having to patch it similar to libmagic, but I’m fine with it either way

Reply:
I'll leave this in then, yes for easier changing of the upstream files one day if needed then.